pkg/server/v3/delta_test.go

```diff
@@ -457,3 +458,95 @@ func TestDeltaCallbackError(t *testing.T) {
 		})
 	}
 }
+
+// BENCHMARKS =====================================================================================================
+
+func BenchmarkDeltaResponseHandlers(b *testing.B) {
+	for _, typ := range testTypes {
+		b.Run(typ, func(b *testing.B) {
+			for n := 0; n < b.N; n++ {
+				config := makeMockConfigWatcher()
+				config.deltaResponses = makeDeltaResponses()
+				s := server.NewServer(context.Background(), config, server.CallbackFuncs{})
+
+				resp := makeMockDeltaStream(&testing.T{})
+				// This is a wildcard request since we don't specify a list of resource subscriptions
```

Comment:
Seems like this is not a wildcard since we set a resource on L479?

Reply:
Oops yup copy pasta, will remove that.